packages/util-dev/src/mockProviders/mockHandleProvider.ts

```diff
@@ -0,0 +1,77 @@
+import { Cardano } from '@cardano-sdk/core';
+
+export const getAliceHandleProviderResponse = {
```

Comment:
Do we want to use those stub responses in Lace repo (for development/tests/storybook etc.)?
- If the answer is yes, then we should probably implement export a StubHandleProvider impl as well.
- If the answer is no, then we should probably hoist this to `cardano-services-client/test/util.ts`, as without the provider implementation I don't see other use cases for it. Are there any?

Reply:
> I will move it to test/utils then.

Please do, thanks